script/Ciel/Day 14/14_01B_03_CIEL14B_5.txt

```diff
@@ -433,52 +524,61 @@ Like a solemn theologian, he asks where my heart lies.
 [sha:ab788b81c544f90ac7055a33fc7f8b6795900d09]{
 -- Page 198, Offset 40835.
 -- 「もう何度も折れてる。直りが早いだけだ」
-"It's been broken many times already. It just heals real quick."
+"It's been broken a whole bunch of times.
+ It just heals real quick."
 }
 [sha:b644dcb60f42e40919a5c37ad3d5069b13c43c7c]{
 -- Page 198, Offset 40836.
 -- 『その場合、懲りない、というのが正しいな』
+%{i}
 "Then perhaps it's safe to say that you just don't learn."
+%{/i}
 }
 [sha:af18407b4a234353cfc1fb3a7e97b68cd9baed8a]{
 -- Page 198, Offset 40837.
 -- は。まったくその通り。
-Heh. That's exactly right.
+Heh. You can say that again.
 }
 [sha:cadc22311febb53e383f27b5136344f055191870]{
 -- Page 199, Offset 40838.
 -- 「いいから、いいかげん力を貸せ。俺が死んだら困るんだろ、おまえも」
-"Stop wasting my time and help me already. It would be bad for you too if I die, right?"
+"Come on, hurry up and help me already.
+ It'd be a problem for you if I died, right?"
 }
 [sha:1d1308071c41e039aec83f5025f5e27aac0877b8]{
 -- Page 199, Offset 40839.
 -- 『私が協力する道理がない。
-"There's no reason for me to cooperate with you.
+%{i}
+"I have no reason to cooperate with you.
+%{/i}
 }
 [sha:de3c215ae4539e525abaef25990cc72dc022ffda]{
 -- Page 199, Offset 40840.
 -- おまえが玉砕しようと姫君が消滅しようと、私の価値に変動はない』
-My values won't change whether you die an honorable death or the princess gets annihilated."
+%{i}
+Regardless of whether or not you die an honorable death or the princess is annihilated,
+ my worth will not change."
+%{/i}
 }
 [sha:1d70b8c86e30251116ba55641ceee01ee688f983]{
 -- Page 200, Offset 40841.
 -- 感情を廃した、実に機械的な回答だった。
-It was a truly mechanical response devoid of emotion.
+It was the emotionless response of a machine.
 }
 [sha:5e6014f4446b372b57a3ed709c516a65418c1f28]{
 -- Page 200, Offset 40842.
 -- ああ、ロアならそう言うだろうさ。
-Yeah, that's the sort of thing Roa would say.
+Yeah, that's the sort of thing Roa would say alright.
 }
 [sha:c53cf03a1a2cda3c0e7be01a549115b0ff40c671]{
 -- Page 200, Offset 40843.
 -- 他人に転生する事も、アルクェイドに執着する事も“次に”持ち越せばいいと。
-It's like he's saying that he will carry his reincarnation into his next life along with his attachment to Arcueid.
+He doesn't care so long as he can perpetuate his reincarnation and his attachment to Arcueid.
```

Comment:
From a logical perspective this doesn't make much sense, but I have a hard time figuring out what was meant